src/core/InputDropdown/styles.ts

```diff
@@ -72,21 +73,35 @@ const minimal = (props: InputDropdownProps): SerializedStyles => {
         color: ${palette?.text?.primary};
       }
       path {
-        fill: ${palette?.text?.primary};
+        fill: ${colors?.primary[400]};
       }
     }
   `;
 };
+
+const isOpen = (props: InputDropdownProps): SerializedStyles => {
+  const palette = getPalette(props);
+  return css`
+    span {
+      color: ${palette?.text?.primary};
```

Comment:
is this color black?

Reply:
the designs actually are not consistent within the design system ref doc. but for [inputdropdown](https://www.figma.com/file/EaRifXLFs54XTjO1Mlszkg/Science-Design-System-Reference?node-id=1876%3A21139), the user input state shows them both as black (the carat should only be colored on active). For [filters](https://www.figma.com/file/EaRifXLFs54XTjO1Mlszkg/Science-Design-System-Reference?node-id=3139%3A19517) (which, iirc, should make use of the inputdropdown, so they should be the same).

I'm checking in with SDS about how the inputdropdown component should actually style it in the default application (then aspen will get it for free).